test.js

```diff
@@ -35,6 +35,18 @@ test('.flagToScope(flag)', async () => {
   assert.equal(starryNight.flagToScope('whatever'), undefined)
 })
 
+test('.createStarryNight with options', () => {
+  assert.rejects(
+    async () => {
+      await createStarryNight(common, {
+        getOnigurumaUrlFs: () => new URL('file:///foo/baz/onig.wasm')
+      })
+    },
+    /no such file or directory, open '\/foo\/baz\/onig.wasm'/,
```

Comment:
```suggestion
    /no such file or directory/,
```

In my experience, Node frequently shows different internal C calls in their error messages, across versions and platforms. Better to not include them.

Reply:
Ok no problem, 63eb1bd removes the path part